extensions/amp-accordion/1.0/accordion.js

```diff
@@ -397,16 +415,47 @@ export function AccordionContent({
 }) {
   const ref = useRef(null);
   const hasMountedRef = useRef(false);
-  const {contentId, headerId, expanded, animate, setContentId} = useContext(
-    SectionContext
-  );
+  const {
+    contentId,
+    headerId,
+    expanded,
+    animate,
+    setContentId,
+    expandHandler,
+    experimentDisplayLocking,
+  } = useContext(SectionContext);
   const classes = useStyles();
+  const [contentVisibility, setContentVisibility] = useState(false);
+  const useDisplayLocking = contentVisibility && experimentDisplayLocking;
 
   useEffect(() => {
     hasMountedRef.current = true;
     return () => (hasMountedRef.current = false);
   }, []);
 
+  useEffect(() => {
+    if (!ref.current) {
+      return;
+    }
+    const element = ref.current;
+    const win = element.ownerDocument.defaultView;
```

Comment:
Does this need `toWin`?

Reply:
Hmm, I think it does not in this case.  My understanding based on the comments of `toWin` is that it is primarily used for casting `Window|undefined` to `Window` since many amp internal functions require the `Window` type.  Whereas in this case, we are not calling any of these amp internal functions.  Let me know if I am missing something though!

Reply:
This should be either `toWin` or `if (!win) return`.

Reply:
The issue is if it is `undefined`, calling `undefined.CSS.supports` will throw.

Reply:
Updated to `if (!win) return`.